### PR TITLE
axios 1.0.0

### PR DESCRIPTION
**Pull request by andreatamezm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +82,7 @@ For frontend use, [you must specify a `proxy` url and provide credentials on the
 For backend use in node or other environments, you can provide [Genability API credentials](https://developer.genability.com/quick-start/) to the client in several ways. The client will search for credentials in the following order:
 
 1. Credentials explicitly provided to the API client
-2. Credentials stored as environment variables GEN_APP_ID and GEN_APP_KEY
+2. Credentials stored as environment variables GENABILITY_APP_ID and GENABILITY_APP_KEY
```
Comment (andreatamezm) [inline README.md:85]: [project is expecting ](https://github.com/Genability/genability-js/blob/437a2ce09567b290e82e6442862c799bad2fcf3e/src/rest-client/credentials.ts#L40)`GENABILITY_APP_ID` & `GENABILITY_APP_KEY`, not `GEN_APP_ID` & `GEN_APP_KEY`

File: src/rest-client/client.test.ts [hunk 2]
```diff
@@ -1,32 +1,15 @@
-const axiosInstanceMock = {
-  interceptors: {
-    request: {
-      use: (): object => {
-        return {};
-      }
+jest.mock('axios', () => ({
+  create: () => ({
```
Comment (rchristy0) [inline src/rest-client/client.test.ts]: Github is throwing a warning here. `Missing return type on function`

**Review by rchristy0 — APPROVED** [review]
This PR change looks fine to me. There is just one warning from github actions. Take a look at that, but if thats fine you can merge. 

Lets just make sure to open a ticket for those test failures. I think the test expectations are probably still accurate, and we should take a closer look at the smart prices endpoint.

**Comment by andreatamezm** [comment]
Here is the ticket for test failures [GEN-3944](https://arcadiapower.atlassian.net/jira/software/c/projects/GEN/boards/138?assignee=712020%3A7b56124f-b427-44ca-a676-5e7e59e20a80&selectedIssue=GEN-3944)

[GEN-3944]: https://arcadiapower.atlassian.net/browse/GEN-3944?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Pull request merged**